### DISCUSSION TITLE
Open Imageglass when SD card is launched into card reader

### DISCUSSION THREAD

**Discussion by ansgar75** [d2phap/ImageGlass#882] [General]
Hi,

I'm using IG for a while. It is a very good tool.
Is there a option to add IG to the Autoplay of Win10 after a SD card is launched into card reader?

Thank you so much.
Ansgar

**Comment by d2phap** [comment]
hi @ansgar75 

I'm not sure if it works but I think you can try setting IG as default photo viewer.
